app-frontend/config/webpack/environments/production.js

```diff
@@ -6,6 +6,7 @@
 
 const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack_fail = require('webpack-fail-plugin');

```

Comment:
Nitpick, but I think we'll want to keep the camel case variable name format going with something like `WebpackFailPlugin` here.